### ISSUE TITLE
FAQ is not reachable by keyboard

### ISSUE THREAD

**Issue by EmmaDawsonDev** [StemWarriorHacks/StemWarriorHacks-Website#7]
I tested your site with just my keyboard and I was unable to reach the questions inside FAQ section to open them. Divs are not automatically tabbable but can be added into the tab order by using tabindex="0" on all divs that should be tabbable.
